web/controllers/users_controller.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Notox.UserController do
+  use Notox.Web, :controller
+  alias Notox.User
+
+  def index(conn, _params) do
+    users = Repo.all(User)
+    render conn, :index, users: users
+  end
+
+  def show(conn, %{"id" => id}) do
+    user = Repo.get(User, id)
+    render conn, :show, user: user
+  end
+
+  def new(conn, _params) do
+    changeset = User.changeset(%User{})
+    render conn, :new, changeset: changeset
+  end
+
+  def create(conn, %{"user" => user_params}) do
+    changeset = User.registration_changeset(%User{}, user_params)
+    case Repo.insert(changeset) do
+      {:ok, user} ->
+        conn
+        |> Notox.Auth.login(user)
+        |> put_flash(:info, "User created!")
+        |> redirect(to: user_path(conn, :index))
+      {:error, changeset} ->
+        render(conn, :new, changeset: changeset)
+    end
+  end
+
+  def edit(conn, %{"id" => id}) do
```

Comment:
Ahh ok. So you're suggesting that I get rid of the `users#edit` action altogether? I understand your point, but I'd like to hear more about the reasoning behind separating a `user` from the user's `profile`. Maybe we can talk about this some more.